src/tests/base/test_api.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from chaosinventory.authentication.models import Token, User
+
+
+class ApiTestCase(TestCase):
+    def setUp(self) -> None:
```

Comment:
should be `set_up`.

Reply:
See https://github.com/chaosinventory/chaosinventory/pull/77#discussion_r650538590